examples/rbac/client/feature_repo/test.py

```diff
@@ -18,8 +18,16 @@ def run_demo():
         print("\n--- Historical features for batch scoring ---")
         fetch_historical_features_entity_df(store, for_batch_scoring=True)
 
-        print("\n--- Load features into online store ---")
-        store.materialize_incremental(end_date=datetime.now())
+        try:
+            print("\n--- Load features into online store/materialize_incremental ---")
+            feature_views= store.list_feature_views()
+            if not feature_views:
+                raise PermissionError("No access to feature-views or no feature-views available.")
+            store.materialize_incremental(end_date=datetime.now())
+        except PermissionError as pe:
+            print(f"Permission error: {pe}")
+        except Exception as e:
+            print(f"An occurred while performing materialize incremental: {e}")
```

Comment:
it is hard to monitor this exception log during the test run. Is there any reason why we are catching and doing nothing?

Reply:
Good point.

I am trying to show the error message incase if the user have no access on feature_view for example unauthorized user. Otherwise it goes and do materialization with zero feature view, will not show any error and give impression that authorize-user run the materialization.
For example see this logs. https://gist.github.com/redhatHameed/9754caa678345456365ea5f5790b06bd#file-logs-L9

Reply:
with this PR it will shows like
```--- Load features into online store/materialize_incremental ---
Permission error: No access to feature-views or no feature-views available.```